Add secret cache doc (#2019)
dapr/docs: daprdocs/content/en/operations/components/setup-secret-store.md

    value: "[aws_session_token]"
```

## Cache for secret stores

> For performance purposes, dapr provides a caching feature for the secret store.

Reviewer: ```suggestion
For performance purposes, Dapr provides a caching feature for the secret store.
```

Author: done

Cache for secret stores can easily be configured by metadata configuration. Each secret store can have its own cache configuration.

### Spec metadata fields

| Field              | Required | Details | Example |
|--------------------|:--------:|---------|---------|

> | cacheEnable               | N        | Enable cache for this secret store or not. Defaults to `false`.| `true`、`false`

Reviewer: ```suggestion
| cacheEnable               | N        | Enable cache for this secret store. Defaults to `false`.| `true`、`false`
```

Author: done

| cacheTTL               | N        | TTL for cache items. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that will be processed as milliseconds. Valid time units are “ns”, “us” (or “µs”), “ms”, “s”, “m”, “h”. Defaults to `5m`  | `5000`、`15m`

> | cacheMemoryLimit               | N        | Maximum length in bytes of the memory usages for cache. Since the value in memory is encrypted, so the memory usage will be lagger than the original secret bytes. When exceeds the limit old items will be evicted. Defaults to `10485760` (1M).  | `10485760`

Reviewer: ```suggestion
| cacheMemoryLimit               | N        | Maximum length in bytes of the memory usages for cache. Since the value in memory is encrypted, the memory usage will be larger than the original secret bytes. When it exceeds this limit, old items will be evicted. Defaults to `10485760` (1M).  | `10485760`
```

Author: done

> Below is an example of secret store configuration with cache enabled.

Reviewer: ```suggestion
Below is an example of a secret store configuration with cache enabled.
```

Author: done
